client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel-mobile/incident-info-panel-mobile.component.html

```diff
@@ -195,14 +189,15 @@ <h1>Evacuations</h1>
                         <div class="subtitle">
                             <img class="icon" height="16" width="16" src="/assets/images/svg-icons/issuing_agency.svg"
                                 alt="issuing_agency" />Issued by {{item.issuingAgency}}
-                            <img height="20" (click)="navigateToEvac(item)" width="20" class="arrow-button" style="background: var(--reds-red-6, #FBE3E3);"
+                            <img height="20" (click)="navigateToEvac(item)" width="20" class="arrow-button"
+                                style="background: var(--reds-red-6, #FBE3E3);"
                                 src="/assets/images/svg-icons/forward-arrow-red.svg">
                         </div>
                     </div>
                 </div>
-                <div *ngIf="mobileEvacAlerts && mobileEvacAlerts.length >= 1">
-                    <div class="evac-title"><img class="icon" src="/assets/images/svg-icons/custom_warning.svg"
-                            alt="ustom_warning" />Evacuation Alerts</div>
+                <div id="evac-alert" *ngIf="mobileEvacAlerts && mobileEvacAlerts.length >= 1">
+                    <div class="evac-title"><img class="icon" style="top: 3px !important" src="/assets/images/svg-icons/custom_warning.svg"
```

Comment:
Same with this one. can the style be added to the class? Or should there be a new class?

Reply:
Done